SUMMARY.md

```diff
@@ -11,6 +11,7 @@
     * [Authentication](up-and-running/auth/README.md)
       * [Open Authentication](up-and-running/auth/open.md)
       * [Github Enterprise Integration](up-and-running/auth/github.md)
+  * [Uninstalling npm Enterprise](up-and-running/uninstalling.md)
```

Comment:
I'd be tempted to put this under `Troubleshooting` , or `Tutorials`:

* I don't think it's good to have the `uninstall` instructions in the `installation` header.
* also, this is something we don't want too many people to do, so I'm fine with it being a little buried in the instructions.